### PR TITLE
fix: Build tree shakeable packages

### PR DESCRIPTION
**Pull request by selbekk**
This commit introduces a second build step, where files now are built
without transpiled imports and exports. This, in addition to the
"sideEffects: false" flag and "module: es/index.js" additions to the
respective `package.json` files, should be enough to satisfy Webpack's
requirements for tree shaking and dead code elimination - leading to
smaller bundles!

### CODE REVIEW

**Review by henrikhermansen — APPROVED** [review]
I don't know exactly what this flag and the es module does, but I assume you know what you're doing there.
From what I can tell it seems you've done a consistent and thorough job. Very nice!

**Comment by selbekk** [comment]
It's already being done for ffe-icons-react, actually - this just applies the same settings and build to the rest of our js packages.

**Review by eriksalhus — APPROVED** [review]

**Review by ksawerysb1 — APPROVED** [review]

**Comment by ksawerysb1** [comment]
I tried using this branch with project code and it worked just fine. :+1:

**Comment by selbekk** [comment]
Good stuff! @antidecaf and @kwltrs - do you want to have a look before I merge this? It's affecting all packages, so I'd love for you to check off on this.

**Comment by ivarni** [comment]
Wait, there's a way to `npm link` stuff now to facilitate trying things out before merging and publishing?

**Comment by selbekk** [comment]
> Wait, there's a way to npm link stuff now to facilitate trying things out before merging and publishing?

Yeah I think lerna fixed the npm link stuff a while back.

**Comment by ksawerysb1** [comment]
> Wait, there's a way to `npm link` stuff now to facilitate trying things out before merging and publishing?

Yes you can use "yarn link" in the designsystem package folder 
```
cd packages/ffe-formatters
yarn link
```

then in the project you want to use the link:
`yarn link @sb1/ffe-formatters`

**Comment by ivarni** [comment]
Well that's good news that they fixed it. Out of all the issues with lerna that was by far the most annoying.

**Review by kwltrs — APPROVED** [review]
Approve, but not sure if lerna will include this commit to all CHANGELOGs.

**Pull request merged**
